File: JavaNIDE/volume/app/src/main/java/sample/For3.java
package sample;
import java.util.*;
class For3
 {
static int i,j,prod=1,n=5;
 static float sum=0;
 public static void main(String[] ar)
 {
  System.out.print("enter last no");
 Scanner m=new Scanner(System.in);
 n=m.nextInt();
  for(i=1;i<=n;i++)
  {
   for(j=1;j<=i;j++)
   {
    System.out.print(j);
    if(j<i)
    {
     System.out.print("*");
    }
    prod=prod*j;
   }
   System.out.print(" + ");
   sum=sum+prod;
   prod=1;
  }
 System.out.print("sum of seriese is"+sum);
}
}

File: CppCompiler/for3.cpp
#include<iostream>
using namespace std;

int main()
{
 int i,sum=0,j;
for(i=1;i<=10;i++)
{
for(j=1;j<=10;j++)
{
if(j%2==0)
{
continue;
}
cout<<j<<endl;
sum=sum+j;
}
cout<<endl;
}
cout<<sum;
return 0;
}